sinabs/backend/dynapcnn/config_builder.py

```diff
@@ -30,7 +35,12 @@ def get_default_config(cls):
 
     @classmethod
     @abstractmethod
-    def build_config(cls, model: "DynapcnnNetwork", chip_layers: List[int]):
+    def build_config(
```

Comment:
Can we have layer2core_map before destination map? To match the order of parameters in general?

Reply:
Also, it looks like the way to build the configuration has changed drastically. 
We need to add a line to the changelog and documentation to refer to this breaking change.